docs/unified/novacustom/initial-deployment.md

```diff
@@ -0,0 +1,201 @@
+# Initial Deployment
+
+## Intro
+
+This document is a guide for the initial installation of Dasharo on a supported
+device. It assumes some knowledge about external flashing and is primarily aimed
+at technicians performing initial installation and not for end users of the
+devices.
+
+## Installing Dasharo
+
+### Initial Installation
+
+=== "12th Gen (Alder Lake)"
+
+    During the initial installation of Dasharo, you should deploy the supported
+    Intel ME version (and configuration) on the device. Since vendor firmware
+    has enabled Intel Boot Guard and BIOS Guard, it is not possible to do this
+    from within the operating system and external flashing of the whole flash
+    chip using a programmer like the CH341a or
+    [3mdeb RTE](https://docs.dasharo.com/transparent-validation/rte/v1.1.0/flashing-guide/#ns5x7x-12th-generation)
+    is required.
+
+    > Publicly released binaries do not contain ME binary. If you need an Intel ME
+    > update for your device, contact us via already established commercial support
+    > channel.
+
+    ### Preparation
+
+    Install flashrom:
+
+    ```bash
+    apt update
+    apt upgrade
+    sudo apt -y install flashrom
+    ```
+
+    ### BIOS installation
+
+    Steps for installing Dasharo BIOS:
+
+    1. Remove the bottom cover of the laptop.
+
+        === "NS5x / NS7x"
+            ![ns5x chips](../../images/ns50mu_board_chips.jpg)
+
+        === "NV4x"
+            ![nv4x chips](../../images/nv4x_board_chips.jpg)
+
+    1. Disconnect the primary battery. (1)
+    1. Disconnect the CMOS battery. (2)
+    1. Attach a WSON-8 probe to the SPI flash chip. (3)
+    1. Backup the current firmware, in case you want to be able to restore it
+       at some point:
+
+        ```bash
+        flashrom -p ch341a_spi -r bios_backup.rom
```

Comment:
In current state, I doubt anyone would use RTE as it is only once menionetd in the link and we use ch341 everywhere else. Maybe we should 2 sections with different programmers. This one with CH341 and another one with RTE, explaining what is the difference (i.e. provide the expected reading/flashing/verification time).

Reply:
Removed mention of RTE for now. Need to figure out how to integrate a separate RTE section smoothly